seqr/views/utils/gene_utils.py

```diff
@@ -1,45 +1,21 @@
-from seqr.views.utils.json_utils import _to_camel_case
+from reference_data.models import GeneInfo
+from seqr.views.utils.orm_to_json_utils import get_json_for_genes, get_json_for_gene
 from xbrowse_server.mall import get_reference
 
-# TODO create new reference data handler for seqr
 
-
-def get_gene(gene_id):
-    reference = get_reference()
-    gene = reference.get_gene(gene_id)
-    gene['expression'] = reference.get_tissue_expression_display_values(gene_id)
-    return _gene_json(gene)
+def get_gene(gene_id, user):
+    gene = GeneInfo.objects.get(gene_id=gene_id)
+    gene_json = get_json_for_gene(gene, user=user, add_notes=True)
+    # TODO load expression data into postgres reference_data
```

Comment:
I think expression data should go into ES. 
It will only ever be queried by gene id, and would significantly affect postgres backup size.

Reply:
great point. pg_dump does have --exclude-schema=

Reply:
One thing I forgot to add - where do you think is the right place to put add redis caching?

Reply:
well, does redis really improve performance over standard postgres queries? Also, probably better to have this conversation elsewhere since its not so relevant to these changes